src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { taskStatusToCopy } from "constants/task";
+import { TaskStatus } from "types/task";
+
+interface Props {
+  status: TaskStatus;
+  count: number;
+}
+export const GroupedTaskStatusBadge: React.FC<Props> = ({ count, status }) => (
+  <BadgeContainer css={statusToColorVariant[status]}>
+    <Number css={statusToColorVariant[status]}>{count}</Number>
+    <Status css={statusToColorVariant[status]}>
+      {status === TaskStatus.Succeeded || status === TaskStatus.SetupFailed
```

Comment:
I think this component should be able to accept any status not only succeeded and failed so that it can be reused in other places. (Build Variant squares, My Patches table)

Reply:
Line 16 is for conditional rendering of plural modifier for "Success" => "Successes" and "Setup Failure" => "Setup Failure". Other statuses such as "Running" and "Failed" don't need the plural modifier. Should I take this out and do it before the return statement? The nested conditional rendering may be confusing. :)

Reply:
Just pushed another method of doing this! :)